tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ng.html

```diff
@@ -168,7 +168,7 @@
     </table>
   </ng-template>
 </div>
-<ng-container *ngIf="selectedTime">
+<ng-container *ngIf="isDataTableEnabled && selectedTime">
```

Comment:
Eventually this is an OR condition?
Or we can build this on its own and completely remove selectedTime

Reply:
Yea eventually we remove selectedTime however, for now we need to have selectedTime here because I use the startStep for the table.